webapp/src/components/sidebar/ModelLibrary.vue

```diff
@@ -210,11 +212,34 @@ function sortModels(a: ModelMetadata, b: ModelMetadata): number {
 }
 
 
-function deleteModelConfirm(modelID: string){
-  toaster.default(`Click here to confirm the model deletion`, {
-    onClick: () => deleteModel(modelID),
-    duration: 10000,
-  });
+async function deleteModelConfirm(modelID: string){
+  const modelInfo = models.value.find(([id]) => id === modelID);
+  const model = await memory.getModel(modelID);
+  if (!modelInfo) return
```

Comment:
```suggestion
  if (modelInfo === undefined) return
```
It's always better to triple equal check when testing undefined, JS has a lot of twisted logic with booleans